biomage/stage/stage.py

```diff
@@ -438,17 +438,17 @@ def stage(token, org, deployments, with_rds, auto):
         click.style(
             "✔️ Deployment submitted. You can check your progress at "
             f"https://github.com/{org}/iac/actions. When the deployment is done"
-            " run the following command to trigger flux synchronization and "
-            " speed up the process:",
+            " you will have to run the following command for each of the repositories "
+            " to trigger flux synchronization and speed up the process:",
             fg="green",
             bold=True,
         )
     )
 
     click.echo()
     click.echo(
-        "\tfluxctl sync --k8s-fwd-ns flux --context arn:aws:eks:eu-west-1:"
-        "242905224710:cluster/biomage-staging",
+        "\tflux reconcile helmrelease <repo> --namespace <repo>-<sandboxId> "
```

Comment:
how about printing the 4 commands so we can just copy paste them and run all if needed?